spec/controllers/accounts_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe RegistrationsController, type: :controller do
+  describe "GET new" do
+    it "assigns a new User to @user" do
+      get :new
+      expect(assigns(:user)).to be_a_new(User)
+    end
+
+    it "renders the new template" do
+      get :new
+      expect(response).to render_template(:new)
+    end
+
+    it "returns a successful response" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "POST create" do
+    let(:valid_params) do
```

Comment:
let, let! và let it be khác nhau thế nào cu

Reply:
Dạ let với let! nó sẽ take care việc tính toán giá trị khi mà init, với let thì giá trị của nó sẽ không được tính toán nếu nó không được gọi, còn let! thì ngược lại ạ, với let_it_be thì nó sử dụng cho những immutable object ạ